script/DeployMaxVault.s.sol

```diff
@@ -139,7 +138,7 @@ contract DeployMaxVault is BaseScript {
             assets[2] = contracts.BNBX();
             setDepositAssetRule(vault, contracts.YNBNBK(), assets);
             setWithdrawAssetRule(vault, contracts.YNBNBK(), assets);
-            setApprovalRule(vault, contracts.YNBNBK(), assets);
+            // TODO: fix approval rule for ynbnbk
```

Comment:
I didn't want to complicate this script since we will be deploying without ynbnbk anyway and configure it separately.